Kitodo/src/main/java/de/sub/goobi/forms/LoginForm.java

```diff
@@ -252,17 +252,16 @@ public String BenutzerkonfigurationSpeichern() {
 
     private void AlteBilderAufraeumen() throws IOException {
         /* Pages-Verzeichnis mit den temporären Images ermitteln */
-        URI myPfad = serviceManager.getFileService()
-                .getInternUri(new File(ConfigCore.getTempImagesPathAsCompleteDirectory()).toURI());
+        URI path = ConfigCore.getTempImagesPathAsCompleteDirectory();
 
         /* Verzeichnis einlesen */
         FilenameFilter filter = new FileNameEndsWithFilter(".png");
-        ArrayList<URI> dateien = serviceManager.getFileService().getSubUris(filter, myPfad);
+        ArrayList<URI> files = serviceManager.getFileService().getSubUris(filter, path);
 
         /* alle Dateien durchlaufen und die alten löschen */
-        if (dateien != null) {
-            for (URI aDateien : dateien) {
-                URI file = myPfad.resolve(aDateien);
+        if (files != null) {
+            for (URI aFiles : files) {
```

Comment:
If you found a better variable name as `aFiles` (previous `aDateien` was even not better) then rename this variable.